sudo/settings.py

```diff
@@ -45,3 +45,6 @@
 # The name of the session attribute used to preserve the redirect destination
 # between the original page request and successful sudo login.
 REDIRECT_TO_FIELD_NAME = getattr(settings, 'SUDO_REDIRECT_TO_FIELD_NAME', 'sudo_redirect_to')
+
+# The name of the view function for reversing
+SUDO_VIEW_NAME = getattr(settings, 'SUDO_VIEW_NAME', 'sudo.views.sudo')
```

Comment:
What about doing `SUDO_REDIRECT_URL` and letting it be the actual url instead of reversing it? We can use `reverse_lazy('sudo.views.sudo')` as the default value.

This is a bit more inline with what Django does, see `LOGIN_REDIRECT_URL`.